parsers/thueringen.py

```diff
@@ -0,0 +1,148 @@
+from urllib.request import urlopen
+from bs4 import BeautifulSoup as parse
+import re
+from utils import Parser
+from datetime import datetime, timedelta
+from pyopenmensa.feed import LazyBuilder
+
+amount_regex = re.compile('\d{1,2},\d{1,2}')
+employees_fee_default = -1
+guests_fee_default = -1
+
+
+def parse_start_date(document):
+	week_start_end_date_regex = re.compile('\d{1,2}\.\d{1,2}\.\d{4}')
+	calendar_week_regex = re.compile('\d{4}-\d{1,2}$')
+
+	option = document.find("option", value=calendar_week_regex, selected=True)
+
+	start_date_str = week_start_end_date_regex.search(option.text)
+
+	return datetime.strptime(start_date_str.group(), '%d.%m.%Y')
+
+
+def parse_fees(document):
+	fees_regex = re.compile('Bedienstete.*')
+
+	fees_p = document.find_all('p', {'class': 'MsoNormal'})
+
+	for fee_candidate in fees_p:
+		fee_strings = fees_regex.findall(fee_candidate.text)
+		for amount_candidate in fee_strings:
+			amount_strings = amount_regex.findall(amount_candidate)
+
+			if len(amount_strings) != 2:
+				continue
+
+			employees_fee = float(amount_strings[0].replace(',', '.'))
+			guests_fee = float(amount_strings[1].replace(',', '.'))
+			return employees_fee, guests_fee
+
+	return employees_fee_default, guests_fee_default
+
+
+def parse_meals(day):
+	ingredients_regex = re.compile('Inhalt:.*')
+
+	meals = []
+	meals_t_rows = day.find_all('tr')
+
+	for meal_data in meals_t_rows:
+		meal_t_datas = meal_data.find_all('td')
+		if len(meal_t_datas) is not 3:
+			continue
+
+		notes = [meal_t_datas[0].text]
+
+		ingredients = ingredients_regex.findall(meal_t_datas[1].text)
+		if len(ingredients) > 0:
+			notes.append(ingredients[0].strip())
```

Comment:
You should add a single note for every ingredient. Also please parse the table at the bottom of the page to write out the abbreviations in full. There are some helpers for this in https://github.com/mswart/pyopenmensa/blob/7b6e7a70ab63530af27b985019f28863f8b8f368/feed.py#L182-L246 that can parse ingredient tables and automatically replace the abbreviations. I can also help you out with this if you want.